### ISSUE TITLE
Should we recommend setting helpertexts on CustomField subfields?

### ISSUE THREAD

**Issue by rolfsmeds** [vaadin/docs#1569]
From TetraLogical's accessibility review report:

> "Although they have accessible names provided via the aria-label attribute, the individual fields lack visible text labels."

While we don't want to recommend applying standard labels on sub-fields, as that would look awful, perhaps we could recommend applying individual helpertexts to each subfield? This would provide visible labels for sighted users. (Users relying on screen readers can already be sufficiently served by aria-labels set on the subfields.)

**Comment by rolfsmeds** [comment]
We should in any case recommend setting `aria-label`s on the subfields.
